Adding Humata case study (#29623)
supabase/supabase: apps/www/_customers/humata.mdx

---
name: Humata

> title: Humata Scales with Supabase: Achieving 4X Cost Savings and Enhanced Performance

Reviewer: ```suggestion
title: Humata Scales with Supabase - Achieving 4X Cost Savings and Enhanced Performance
```

# Use meta_title to add a custom meta title. Otherwise it defaults to '{name} | Supabase Customer Stories':

> meta_title: Humata Scales with Supabase: Achieving 4X Cost Savings and Enhanced Performance

Reviewer: ```suggestion
meta_title: Humata Scales with Supabase - Achieving 4X Cost Savings and Enhanced Performance
```

description: By partnering with Supabase, Humata achieved a 4X reduction in vector database costs, streamlined their development, and consolidated their data so they could scale seamlessly.
# Use meta_description to add a custom meta description. Otherwise it defaults to {description}:
meta_description: By partnering with Supabase, Humata achieved a 4X reduction in vector database costs, streamlined their development, and consolidated their data so they could scale seamlessly.
author: paul_copplestone
author_title: Supabase
author_url: https://github.com/kiwicopple
author_image_url: https://avatars2.githubusercontent.com/u/10214025?s=400&u=c6775be2ae667e2acae3ccd347fed62bb3f5b3e7&v=4
logo: /images/customers/logos/humata.png
logo_inverse: /images/customers/logos/light/humata.png
og_image: /images/customers/og/humata.jpg
tags:
  - supabase
date: '2024-10-01'
company_url: 'https://www.humata.ai'
stats: []
misc:
  [
    { label: 'Use case', text: 'AI' },
    {
      label: 'Solutions',
      text: 'Supabase Database, Supabase Auth, Supabase Realtime, Supabase Vector',
    },
  ]
about: Use AI to ask questions across all of your files.
---

## Introduction

> Humata, a platform enabling millions of users to chat with and analyse their all documents, relies on cutting-edge technology to support rapid growth and high user demand. Humata started as PDF AI that allows anyone to chat with their files, initially geared for scientific researchers. Humata has now expanded to serve major enterprises and government institutions by connecting their knowledge base to AI and allowing them to get answers they can trust with best-in-class referencing. With a tech stack that includes Supabase, GCP, GPT-4o, Vercel, and NEXT.js, Humata’s platform provides real-time, mission-critical analysis for researchers, enterprises, and government institutions. By partnering with Supabase, Humata achieved a 4X reduction in vector database costs, streamlined their development, and consolidated their data so they could scale seamlessly.

Reviewer: ```suggestion
Humata, a platform enabling millions of users to chat with and analyse their all documents, relies on cutting-edge technology to support rapid growth and high user demand. Humata started as PDF AI that allows anyone to chat with their files, initially geared for scientific researchers. Humata has now expanded to serve major enterprises and government institutions by connecting their knowledge base to AI and allowing them to get answers they can trust with best-in-class referencing. With a tech stack that includes Supabase, GCP, OpenAI, Vercel, and NEXT.js, Humata’s platform provides real-time, mission-critical analysis for researchers, enterprises, and government institutions. By partnering with Supabase, Humata achieved a 4X reduction in vector database costs, streamlined their development, and consolidated their data so they could scale seamlessly.
```

> Humata, a platform enabling millions of users to chat with and analyse their all documents, relies on cutting-edge technology to support rapid growth and high user demand. Humata started as PDF AI that allows anyone to chat with their files, initially geared for scientific researchers. Humata has now expanded to serve major enterprises and government institutions by connecting their knowledge base to AI and allowing them to get answers they can trust with best-in-class referencing.  With a tech stack that includes Supabase, GCP, GPT-4o, Vercel, and NEXT.js, Humata’s platform provides real-time, mission-critical analysis for researchers, enterprises, and government institutions. By partnering with Supabase, Humata achieved a 4X reduction in vector database costs, streamlined their development, and consolidated their data so they could scale seamlessly.

Reviewer 2: Not a blocker, but I feel that there is a good chance referencing the model name `GPT-4o` here would be outdated really soon. Maybe just saying `OpenAI` or something generic might be better in the long run? I'm okay keeping it the current way though, so your call!
